### ISSUE TITLE
[BUG] Rail sounds not stopping

### ISSUE THREAD

**Issue by FlyingDebris** [tsunamayo/Starship-EVO#2595]
### Short Description of the issue:

When using a rail, the sound will continue indefinitely, even after the rail has finished moving.

### Steps to reproduce the issue:

Create rail, move rail via button/switch.

### Environment
- Starship EVO version: 20w27b
- Graphics Card: GTX 1050

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4859418/output_log.txt)

**Comment by tsunamayo** [comment]
Is it on a specific blueprint? I dont think I have this issue. Thanks

**Comment by FlyingDebris** [comment]
It hasn't been tied to a specific blueprint, but it seems to be very intermittent. Reloading fixes the issue.

**Comment by ProPeach** [comment]
I've also noticed the paint sound effect does not stop sometimes, especially after spraying for a long time. It only stops when you start painting again.

**Comment by tsunamayo** [comment]
@ProPeach yes indeed, but that must be a different issue (I think we have a case for that one)

**Comment by TripleHelixxx** [comment]
So I’ve been hesitant to report this because I thought it already had been, but it seems that the paint sound not stopping and the rail sound not stopping are somewhat similar issue, at least based on what I’ve experienced. I can’t reproduce it perfectly, but it seems to always happen when the character is moving. Reloading the game isn’t entirely necessary, you can stand still and tap with a paint, and it stops both the infinite paint sound and the infinite Rail sound.

**Comment by tsunamayo** [comment]
ho I see, then maybe it is the same issue after all...

**Comment by Tezlz** [comment]
I've had this issue for a while as well, the exact same way as @TripleHelixxx described. Hadn't reported it either becuase I couldn't re-create it consistently. It is probably the same issue as #2527 because it occurs under mostly the same circumstances and both can be stopped by briefly spraying anything with paint.

**Comment by FlyingDebris** [comment]
When I was testing, I was unable to get it to stop by spraying, but I may just be a special snowflake in that regard. It seems pretty similar to #2527 as Tezlz mentioned, though.
